doc/usage.rst

```diff
@@ -210,6 +211,7 @@ few command line options:
    to create the virtualenv. It will only work with Python 3.4 or later,
    e.g. by using the option
    :option:`--python` ``/usr/bin/python3.4``.
+   Must not be used with `--setuptools`.
```

Comment:
I think we can drop this, the documentation for `--setuptools` flag should be enough